CHANGELOG.rst

```diff
@@ -16,6 +16,8 @@ Please See the `releases tab <https://github.com/openedx/xblock-lti-consumer/rel
 Unreleased
 ~~~~~~~~~~
 
+* Show a warning in Studio if an LTI 1.1 consumer has an invalid `lti_id`.
```

Comment:
Do you think we should release it?

Reply:
I was thinking it's fine to wait for another release; this is a fairly minor change, and there's no rush.